### PR TITLE
Placeholder for Visual Studio Debugging Documentation

### PR DESCRIPTION
**Pull request by anakrish**
Signed-off-by: Anand Krishnamoorthi <anakrish@microsoft.com>

### CODE REVIEW

File: docs/GettingStartedDocs/Windows_visual_studio.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Debugging ELF Enclaves Using Visual Studio
```
Comment (dthaler) [inline docs/GettingStartedDocs/Windows_visual_studio.md]: I do not think this page should just be about "Debugging".  This should be "Developing" which is much more encompassing.   Here's the page we have in the new_platforms branch:
https://github.com/openenclave/openenclave/blob/feature.new_platforms/new_platforms/docs/visualstudio_dev.md

I plan to update those pages to merge into master, unless we want to move the dev pages out of the main OE project.

Comment (anakrish) [reply docs/GettingStartedDocs/Windows_visual_studio.md]: @dthaler Sure. We want a page to demonstrate debugging using Visual Studio that we can point to from the Visual Studio Release notes. We have a vanity link aka.ms/OpenEnclaveVisualStudio.
I can make that point to your page and then later have a link from your page to OE SGX Debugging page.
@radhikaj @dthaler What do you think? If so, I can close this PR and instead just update the link

Comment (jhand2) [reply docs/GettingStartedDocs/Windows_visual_studio.md]: I don't think there's much value in adding a placeholder page right now if it doesn't provide any information.

Comment (anakrish) [reply docs/GettingStartedDocs/Windows_visual_studio.md]: I have changed the link to point to https://github.com/openenclave/openenclave/blob/feature.new_platforms/new_platforms/docs/VisualStudioWindows.md

**Comment by anakrish** [comment]
Closing this PR. Let me know if you have additional comments.

**Pull request closed without merging**